src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,20 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>{' Posted by  '}</p>
+
+      <UserInfo user={post.user} />
```

Comment:
```suggestion
      {post.user && <UserInfo user={post.user} />}
```
user can be null, so let's add the check